### DISCUSSION TITLE
litecoin lightning network

### DISCUSSION THREAD

**Discussion by redimongo** [btcpayserver/btcpayserver#6405] [Technical Support]
I am having problems turning on the litecoin lighting network with my current set up, I am wondering if I have done something wrong?

```
Parameters passed:
BTCPAY_PROTOCOL:https
BTCPAY_HOST:crypto.domain.com
BTCPAY_LIGHTNING_HOST: 
BTCPAY_ADDITIONAL_HOSTS:
REVERSEPROXY_HTTP_PORT:80
REVERSEPROXY_HTTPS_PORT:443
REVERSEPROXY_DEFAULT_HOST:none
LIBREPATRON_HOST:
ZAMMAD_HOST:
WOOCOMMERCE_HOST:
BTCTRANSMUTER_HOST:
CHATWOOT_HOST:
BTCPAY_ENABLE_SSH:true
BTCPAY_HOST_SSHKEYFILE:
LETSENCRYPT_EMAIL:
NBITCOIN_NETWORK:mainnet
LIGHTNING_ALIAS:
BTCPAYGEN_CRYPTO1:btc
BTCPAYGEN_CRYPTO2:ltc
BTCPAYGEN_CRYPTO3:
BTCPAYGEN_CRYPTO4:
BTCPAYGEN_CRYPTO5:
BTCPAYGEN_CRYPTO6:
BTCPAYGEN_CRYPTO7:
BTCPAYGEN_CRYPTO8:
BTCPAYGEN_CRYPTO9:
BTCPAYGEN_REVERSEPROXY:nginx
BTCPAYGEN_LIGHTNING:clightning
BTCPAYGEN_ADDITIONAL_FRAGMENTS:opt-save-storage-xs;opt-add-thunderhub;opt-lnd-keysend
BTCPAYGEN_EXCLUDE_FRAGMENTS:nginx-letsencrypt
BTCPAY_IMAGE:
ACME_CA_URI:production
TOR_RELAY_NICKNAME: 
TOR_RELAY_EMAIL: 
PIHOLE_SERVERIP: 
FIREFLY_HOST: 
```

**Comment by pavlenex** [comment]
Hard to tell, what's the actual problem that you're having?

**Comment by mp21-org** [comment]
To say it with the words from @Kukks [here](https://chat.btcpayserver.org/btcpayserver/pl/nymz33iykpnody44fx6optrb7a):
> Literally noone uses lightning on litecoin.  You can take a look at the commit that removed it here: https://github.com/btcpayserver/btcpayserver-docker/commit/867f34ac3c55dc0509421532c5dfdb70b952edc4

**Reply by redimongo** [reply]
Well that kinda sucks. Because am looking for away to do L402 with Litecoin. 

But I need to be able to do macaroon and preimages on invoices. 

We already do it with Sats. 

I was able to reinstall it by adding those two files manually and I am just waiting for the nodes to sync and we will see if works.
